web/modules/custom/joinup_core/joinup_core.module

```diff
@@ -233,3 +235,32 @@ function joinup_core_form_alter(&$form, FormStateInterface $form_state, $form_id
     }
   }
 }
+
+/**
+ * Implements hook_ENTITY_TYPE_view().
+ *
+ * Shows an information message to users who have access to the draft version
```

Comment:
We have discussed with @sandervd and @brummbar and decided that we can have this implemented in the full view at least for now since there are no current plans to show a full view in another page other than the canonical view of the entity.